src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackTool.targets

```diff
@@ -38,6 +38,12 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <NETSdkError Condition="'$(SelfContained)' == 'true'"
                  ResourceName="PackAsToolCannotSupportSelfContained" />
+    
+    <NETSdkError Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'"
+                 ResourceName="PackAsToolOnlySupportNetcoreapp" />
+    
+    <NETSdkError Condition="'$(_TargetFrameworkVersionWithoutV)' &lt; '2.1'"
```

Comment:
Should I access _TargetFrameworkVersionWithoutV or calculate it again. It is created by SDK, but it is a previous process.

hard code 2.1 is fine? it **is** a hard number

Reply:
Better to use 2.1.0 to be absolutely sure msbuild does not coerce to float instead of version.

Reply:
As long as it's in one place only and we have near zero expectation of changing it again, I think hard-coding is actually fine and honestly more readable.